flatfs_test.go

```diff
@@ -687,6 +696,15 @@ func testDiskUsageEstimation(dirFunc mkShardFunc, t *testing.T) {
 	if diff > maxDiff {
 		t.Fatal("expected a better estimation within 5%")
 	}
+
+	// Make sure DiskUsageNotes files exists and the correct value
+	d, err := ioutil.ReadFile(filepath.Join(temp, flatfs.DiskUsageNotes))
+	if err != nil {
+		t.Fatal(err)
+	}
+	if string(d) != "initial-approximate\n" {
```

Comment:
Same

Reply:
It's a test, so I want to make sure it has the right value.